Razor pagination

daniel-fich opened the pull request:
* `.../` is equvalent to `.../?page=1` and shows the first the first page for all cheeps.
* `.../?page=n` shows page number `n`.
* `.../{Author}` is equvalent to `.../{Author}?page=1` and shows the first the first page for each author.
* `.../{Author}?page=n` shows page number `n`.
* Each page shows at most 32 cheeps.
* Page navigation for `.../` and `.../{Author}` is implemented.

Casper2411 reviewed (commented):
Everything looks really good! nice User experience, and understandable code, however, it would maybe be preferable to have a comment describing the new function in the cshtml file, and what is does in a comment.
Also Helge's Cheeps need to be added, because it is checked for on the Bdsa website(http://209.38.208.62/report_razor_apps.html).
Other then that, it looks perfect! Nice job team🤓

daniel-fich commented:
Hello @Casper2411. Thank you very much for the good feedback. I've added a few doc comments and I also added "Helge" as a user to the example cheeps. We could probably add some more comments later, but let's leave it as is for now.
On that note, nice job team 🤓.

Casper2411 commented:
The changes have been made, and everything looks perfect, im so proud of this community.
